### DISCUSSION TITLE
How can I help with Laravel framework

### DISCUSSION THREAD

**Discussion by michaelnabil230** [laravel/framework#42021] [General]
I want to help the Larville community, but I don't know how to do that. Is there a place where you can download the things needed to implement the development or add some things?

**Comment by lk77** [comment]
There is already a ton of contributors for the laravel framework, i don't think it's a good idea, try to find smaller projects that actually needs contributors, it could be third party laravel packages for example

I think you should aim for projects with under 1k stars, projects that are viable and getting traction, but with room for improvements and new contributors.
